cmd/openshift-install/create.go

```diff
@@ -108,6 +110,18 @@ var (
 					logrus.Fatal(err)
 				}
 
+				// FIXME: baremetal platform needs to perform additional steps after bootstrap completes
+				// See https://github.com/openshift-metal3/openshift-installer/issues/60
+				if assetStore, err := assetstore.NewStore(rootOpts.dir); err == nil {
+					installConfig := &installconfig.InstallConfig{}
+					if err := assetStore.Fetch(installConfig); err == nil {
+						if installConfig.Config.Platform.Name() == baremetaltypes.Name {
+							logrus.Warn("Exiting, baremetal platform needs intervention to continue")
+							return
+						}
+					}
+				}
+
```

Comment:
This has now been removed and will be gone in the next PR update: https://github.com/openshift-metal3/kni-installer/pull/136